File: mirror/src/oven/menusm/menus.m
	if (msp->nmenus >= MAX_MENUS)
	    return ((MENUS *)1);
	if ((mp = (MENU *)calloc (1, sizeof(MENU))) == (MENU *)0)
	    return ((MENUS *)0);
	msp->menu[msp->nmenus++] = mp;

File: mirror/src/oven/menusm/menus.c
#include "../menus.h"
#include "menus.ext"

/* menus - initialize the menus database
 */
MENUS *
menus ()
{
	MENUS	*msp;
	MENU	*mp;
	ITEM	*ip;
	char	*calloc();

	if ((msp = (MENUS *)calloc (1, sizeof(MENUS))) == (MENUS *)0)
	    return (msp);

#include "menus.vc"

	return (msp);
}
